src/main/java/io/stargate/sgv2/jsonapi/metrics/TenantRequestMetricsTagProvider.java

```diff
@@ -1,4 +1,4 @@
-package io.stargate.sgv2.jsonapi.api.v1.metrics;
+package io.stargate.sgv2.jsonapi.metrics;
```

Comment:
note for me: this class is duplicating code from TenantRequestMetricsFilter and does not explain WHY it exists What metrics is this tag being added to ? 

we need to kill it or fix it